src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -238,6 +239,7 @@
     "strawberry-graphql": ("strawberry",),
     "streamlit-aggrid": ("st_aggrid",),
     "tensorboard": ("torch.utils.tensorboard",),
+    "UnleashClient": ("UnleashClient"),
```

Comment:
This needs a trailing comma. Otherwise it maps to each character :sweat_smile: 

```suggestion
    "UnleashClient": ("UnleashClient",),
```